app/src/main/java/com/swent/suddenbump/ui/overview/Overview.kt

```diff
@@ -61,7 +63,11 @@ fun OverviewScreen(navigationActions: NavigationActions, userViewModel: UserView
   val groupedFriends by userViewModel.groupedFriends.collectAsState()
 
   // Charge les emplacements des amis lorsque l'écran est composé
-  LaunchedEffect(Unit) { userViewModel.loadFriends() }
+  LaunchedEffect(Unit) {
+    Log.d("OverviewScreen", "currentUser: $currentUser")
+    Log.d("OverviewScreen", "groupedFriends: $groupedFriends")
```

Comment:
Maybe you can remove this log as it's not useful

Reply:
done